src/client/java/net/neoforged/neoforge/client/ClientHooks.java

```diff
@@ -1169,4 +1174,11 @@ public static Overlay createLoadingOverlay(Minecraft minecraft, ReloadInstance r
             return new LoadingOverlay(minecraft, reloadInstance, errorHandler, fadeIn);
         }
     }
+
+    @ApiStatus.Internal
+    public static Set<MetadataSectionType<?>> getDefaultMetadataSectionTypes(Set<MetadataSectionType<?>> vanillaTypes) {
```

Comment:
The method name is really weird for a one-off event.

Reply:
I don't think it's that weird, considering the field it maps to is DEFAULT_METADATA_SECTIONS.

Reply:
Yeah it's kinda fine now that it doesn't fire an event anymore. But still, add that it's sprite or atlas related.